### DISCUSSION TITLE
Not accepting '{' as first character

### DISCUSSION THREAD

**Discussion by walidam** [halaxa/json-machine#79] [Q&A]
Unfortunately, my json is without keys:
{{a:1, b:2},{c:3, d:4}} 
JsonMachin not accept that saying:
```
In Parser.php line 368:
                                            
  Unexpected symbol '{' At position 18139.
```
Is there a solution for that please?

**Comment by walidam** [comment]
Same thing with [{a:1, b:2},{c:3, d:4}]

**Reply by halaxa** [reply]
Your JSON is wrongly encoded. Object cannot miss keys. As for the second example, object keys must be double quoted.
